site/app/templates/admin/UploadWrapperForm.twig

```diff
@@ -2,26 +2,11 @@
 	<h1>Customize Website Theme</h1>
 	<div>
         Instructors can upload simple html files to use as a wrapper around the Submitty website.<br>
-        They can be placed in any of the locations shown in the below diagram.<br>
         Note that they are loaded in frames and will not be able to run scripts or affect the rest of Submitty.<br>
         <br>
         You can also upload a CSS file to override the default styles from Submitty.<br>
         <a target=_blank href="http://submitty.org/instructor/website_customization">More information about Website Customization <i style="font-style:normal;" class="fa-question-circle"></i></a>
     </div>
-    <br>
-    <div>
-        Here is how the site will be laid out:
-        <div class="wrapper-example">
-            <div class="wrapper-top">Top Bar</div>
-            <div class="wrapper-center">
-                <div class="wrapper-left">Left<br>Sidebar</div>
-                <div class="wrapper-body">Submitty</div>
-                <div class="wrapper-right">Right<br>Sidebar</div>
-            </div>
-            <div class="wrapper-bottom">Bottom Bar</div>
-        </div>
-    </div>
-    <br>
```

Comment:
Try to keep this last break tag to have some space between the container and link about website theme information.

Reply:
done